app/views/content_items/manuals/_header.html.erb

```diff
@@ -31,10 +29,8 @@
     <%= render 'govuk_publishing_components/components/metadata', metadata_with_ga4_tracking %>
 
     <div class="in-manual-search">
-      <form action="/search/all" >
-
+      <form action="/search/all" data-module="ga4-form-tracker" data-ga4-form='{ "event_name": "search", "type": "content", "section": "<%= content_item.title %>", "action": "search" }'>
```

Comment:
It might be worth using `.to_json` to build this, as if the `content_item.title` contains a double quote, it looks like the tracking breaks.

E.g.

```HTML
<% test_title = "This has an \" escaped quote in it"%>
      <form action="/search/all" data-module="ga4-form-tracker" data-ga4-form='{ "event_name": "search", "type": "content", "section": "<%= test_title %>", "action": "search" }'>
        <input type='hidden' name="manual[]" value="<%= content_item.base_path %>">
```
Returns this on submit

```JS
GA4 configuration error: Expected ',' or '}' after property value in JSON at position 71 (line 1 column 72) Location {ancestorOrigins: DOMStringList, href: 'http://127.0.0.1:3090/hmrc-internal-manuals/compliance-handbook/ch160200', origin: 'http://127.0.0.1:3090', protocol: 'http:', host: '127.0.0.1:3090', …}
```

Also, the search term is coming through as `[redacted]` - is that correct?

Thanks

Reply:
Good shout, thanks for spotting that, have reworked.

Going to go back to the PAs on the search term - you're probably right, we should should capture it but process it for PII.